spec/sendcloud/operations/sender_addresses_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe Sendcloud::Operations::SenderAddressesRequest do
+  let(:default_base_url) { "https://panel.sendcloud.sc/api/v2/" }
+
+  before do
+    configure_client(base_url: default_base_url)
+  end
+
+  describe "#execute" do
+    let(:subject) { described_class.new }
+
+    it "returns a response in the expected format" do
+      VCR.use_cassette("sender_addresses_request/valid_request") do
+        result = subject.execute
+
+        aggregate_failures do
```

Comment:
Any reason for putting the `aggregate_failures` in a block rather than in the `it`?

eg...

```
it "returns a response in the expected format" aggregate_failures: true do
```